### DISCUSSION TITLE
Natural gas emission factors based on where the gas is produced

### DISCUSSION THREAD

**Discussion by tth4vb** [electricitymaps/electricitymaps-contrib#4420] [Ideas]
Hey all!

I came across this interesting stat in The Economist that LCA co2eq emissions of natural gas varies widely depending on where the gas comes from due to the infrastructure/production process. For example, gas from Russia has 3x higher emissions due to shoddy infrastructure that moves it around (leaking pipes, etc.). This means that countries that import more natural gas from Russia (ex. Germany) have much higher natural gas-related emissions from an LCA perspective than countries who get natural gas from other sources (ex. the Netherlands) @pierresegonne word on the street is you're looking into more granular emission factors, so I thought you in particular would be interested, as this could have a pretty significant impact on overall carbon intensity for some countries (and highlight a critical piece of the puzzle when evaluating the climate impact of natural gas - infrastructure/upstream emissions - I can see a cool blog post coming from team eMap on this to complement the implementation)

Pic from article - looks like source is Rocky Mountain Institute:
![photo_2022-08-06_13-35-22](https://user-images.githubusercontent.com/8129157/183247413-b7f29ea1-304f-4a04-9fc6-fe073fc89767.jpg)

I think I found the source article from RMI here: https://rmi.org/which-gas-will-europe-import-now-the-choice-matters-to-the-climate/

Hope this is helpful, hit me up if I can help :)

- Trevor

**Comment by VIKTORVAV99** [comment]
This is definitely interesting and might be significant enough to try and flowtrace eventually. There has been some discussions around that before but no conclusion was reached as we where unsure if the difference would be significant enough for it to be worth the time and resources needed to do it.

But until we have a decision on that this data could be used for #738 but it seems like it only contain data for Germany. Another issue is that we need to adopt a co2eq model for natural gas that split infrastructure and fuel emissions to get a fair and proper co2eq estimate.

**Comment by pierresegonne** [comment]
That's an interesting article you've found @tth4vb 

Similarly, a super thorough review of well-to-city emissions for gas supplies in China has been published [here in nature](https://www.nature.com/articles/s41467-020-14606-4.pdf).

I think for the time being, given the time constraints (we're supposed to finish a first version by the end of the week), we won't go in depth with well-to-city upstream emissions just yet. 

Additionally, the rapid shifts in supply we're seeing with the war in Ukraine means that it's currently hard to assess where each country is getting its gas from.

It would nevertheless be a very good idea to start gathering data around both the origin of gas per zone, and well-to-city emission factors per region of the world, to improve the accuracy of the lifecycle emissions.

Let's keep this discussion open and expand it :)
